modules/mapper-extras/src/test/java/org/opensearch/index/mapper/SearchAsYouTypeFieldMapperTests.java

```diff
@@ -361,6 +361,9 @@ public void testTermVectors() throws IOException {
 
         PrefixFieldMapper prefixFieldMapper = getPrefixFieldMapper(mapper, "field._index_prefix");
         assertFalse(prefixFieldMapper.fieldType.storeTermVectors());
+        assertFalse(prefixFieldMapper.fieldType.storeTermVectorOffsets());
```

Comment:
Instead of just checking the `yes` index option can we iterate through all `term_vector` index options? `yes`, `with_offsets`, `with_positions_offsets`, `with_positions`, `with_positions_offsets_payloads`, and `with_positions_payloads` and assert the proper `true`/`false` base on the option we're testing?

Reply:
I added tests for all term vectors.